build.sh

```diff
@@ -105,9 +105,17 @@ if (( ${NUMARGS} == 0 )) || hasArg libcudf; then
     make -j${PARALLEL_LEVEL} VERBOSE=${VERBOSE} ${INSTALL_TARGET}
 fi
 
-# Build and install the cudf Python package
+# Build and install the nvstrings and cudf Python packages
```

Comment:
We should add `libnvstrings` and `nvstrings` commands here instead of tying these into the existing installation commands.